schema/course.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$anchor": "vimeo-course",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "$patch": {
+    "source": {
+      "$ref": "course"
+    },
+    "with": {
+      "properties": {
+        "_globals": {
+          "type": "object",
+          "default": {},
+          "properties": {
+            "_vimeo": {
+              "type": "object",
+              "default": {},
+              "properties": {
+                "ariaRegion": {
+                  "type": "string",
+                  "title": "Aria region",
```

Comment:
I think we should prefix the title here `Vimeo:` so it's more specific in the course settings 

<img width="783" alt="Screenshot 2023-02-23 at 16 26 59" src="https://user-images.githubusercontent.com/11569678/220972685-b8597e0d-44f3-43cb-8424-802e4b28d256.png">

Reply:
I'm on board.